core-redirects-studio-plugin/src/main/joo/com/tallence/core/redirects/studio/editor/form/RedirectEditPanel.mxml

```diff
@@ -107,22 +107,31 @@
           </plugins>
         </FieldContainer>
 
-        <ui:BoundRadioGroup width="auto"
```

Comment:
Hast du die RadioGroup durch eine ComboBox ersetzt, da bei der BoundRadioGroup in manchen Fällen der Wert nicht richtig übertragen wurde? Prinzipiell finde ich in diesem Fall eine RadioGroup besser.

Reply:
Aber wieso wird oben beim RedirectType eine ComboBox verwendet und hier eine RadioGroup? Ich finde Radio-Buttons nicht prinzipiell schlecht, aber wir sollten einheitlich bleiben. M.E. gibt es ja UX-technisch gesehen an der Stelle keinen Grund für Radio-Buttons

Reply:
Stimmt, ein guter Einwand. Durch die Radio-Buttons ist ein Klick weniger notwendig, bei nur zwei Einträgen finde ich deswegen Radio-Buttons eigentlich besser. Finde die Lösung jetzt aber auch gut und einheitlich 👍